### DISCUSSION TITLE
Restrict requests to just `GET`

### DISCUSSION THREAD

**Discussion by engnatha** [grafana/grafana-infinity-datasource#676] [Q&A]
Awesome plugin! Really enjoying how flexible it is. I was wondering if there's any way to restrict the drop down to just `GET` requests or if there's any potential value in a feature like that.

**Comment by yesoreyeram** [comment]
I don't know if that really matter. But ideally you want to restrict at your api level permission. ( The api key or user id you are using should have read only permission )
